test/test_board.h

```diff
@@ -84,17 +85,19 @@ class testBoard : public Board
   bool sonar_check(void);
   float sonar_read(void);
 
+// RC
+  void rc_init(rc_type_t rc_type);
+  bool rc_lost();
+  float rc_read(uint8_t channel);
+
 // PWM
-// TODO make these deal in normalized (-1 to 1 or 0 to 1) values (not pwm-specific)
-  void pwm_init(bool cppm, uint32_t refresh_rate, uint16_t idle_pwm);
-  bool pwm_lost();
-  uint16_t pwm_read(uint8_t channel);
-  void pwm_write(uint8_t channel, uint16_t value);
+  void pwm_init(uint32_t refresh_rate, uint16_t idle_pwm);
+  void pwm_write(uint8_t channel, float value);
 
 // non-volatile memory
   void memory_init(void);
   bool memory_read(void *dest, size_t len);
-  bool memory_write(const void *src, size_t len);
+  bool memory_write(void *src, size_t len);
```

Comment:
Should still be const? (see above comment)

Reply:
Probably - I'll have to re-work the SPI driver to fix this.  

The SPI driver is designed to allow reading and writing from the same array (for the IMU message).  Given that this is allowed, I can't currently guarantee the `const` of the written array.  I'll just have to overload the `write` function with the proper `const`-ness